addon_service/migrations/0013_externalcredentials_int_credentials_format_and_more.py

```diff
@@ -0,0 +1,44 @@
+# Generated by Django 4.2.7 on 2025-03-04 15:30
+
+from django.db import (
+    migrations,
+    models,
+)
+
+import addon_service.common.validators
+from addon_service.credentials.models import ExternalCredentials
+
+
+def migrate_credential_format(*args, **kwargs):
+    skipped = 0
+    processed = 0
+    for credential in ExternalCredentials.objects.iterator():
```

Comment:
Is it possible to optimize this, e.g. with `.filter(<...>).update(<...>)`?

Not done benchmarking it locally yet, but a rough estimate for this runtime is a little over an hour for prod.